content/en/docs/concepts/certificate.md

```diff
@@ -62,6 +62,39 @@ which contains the following explanation:
 > assumption that the remote end must already possess it in order to
 > validate it in any case.
 
+{{% alert title="Use ca.crt with caution" color="warning" %}}
+
+**When configuring a client** to connect to a TLS server with a serving certificate that is signed by a private CA,
+you will need to provide the client with the CA certificate in order for it to verify the server.
+If the client is running in a Kubernetes Pod _DO NOT MOUNT THE SAME SECRET AS THE SERVER_,
+to access the `ca.crt` file, because:
```

Comment:
nitpick: the commas don't look quite right to me for this on the preview. Also a minor rewording:

```suggestion
While `ca.crt` will likely contain the certificate you need to trust, _DO NOT MOUNT THE SAME SECRET AS THE SERVER_ so you can access `ca.crt`. This is because:
```